### ISSUE TITLE
hi vasishtbatchu, can you make a version with Input shaping on? thanks

### ISSUE THREAD

**Issue by sightanignat** [vasishtbatchu/Ender-3-Max-Custom-Firmware#1]
I have an ender 3 max, with sprite extruder pro, CRTouch and dual z. Im trying to compile my self a marlin with and got nowhere, heheh then i came a cross with your BIN. it works perfectly. but i can't get the input shaping into play. i guess its turned off. can you make a version with input shaping turned on? thanks a bunch.

**Comment by vasishtbatchu** [comment]
Yeah so imo the best way to compile the firmware is to use vscode with a the needed marlin compilers or Auto Build Marlin, but I understand the amount of troubleshooting it takes to get it to work 😅.

Luckily I got the version of marlin I compiled to work but it did take a lot of weird fixes like using the original cr touch cable instead of the port on the sprite pro. Because of the quirks, I did switch to klipper 6months ago for stuff like input shaping and pressure advance. So I'd reccomend getting a pi and using klipper instead if you have the ability to..

However, I'd love to help and can definitely add the input shaping setting. (I will also add the non compiled files). Is there a certain macro you would prefer to use?

**Comment by sightanignat** [comment]
wow thanks for the reply. the compiled bin is just fine, just please add the input shaping and pressure adv. thanks a lot. and i think ill get my hands on a Ras PI one of these days.

**Comment by sightanignat** [comment]
did you already updated and reupload the firmware? cant wait to try and start to calibrate.
